Makefile

```diff
@@ -25,9 +25,17 @@ env:: | $(CONDA_ENV_PYTHON)
 	git submodule init
 	git submodule update --init --recursive
 	mkdir -p env/symbiflow
+	wget -q https://github.com/QuickLogic-Corp/quicklogic-fpga-toolchain/releases/download/v1.1.0/Symbiflow_v1.1.0.gz.run
+	mkdir env/quicklogic
```

Comment:
Should be `mkdir -p ...`. This way `make env` doesn't fail when run second time.

Reply:
Fixed